package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "notifications-panel",
-  "version": "1.2.1",
+  "version": "1.3.0",
```

Comment:
to be consistent though, we just made this decision in a recent PR along these "wrong" rules